contrib/test_jenkins.sh

```diff
@@ -475,9 +475,11 @@ run_ucx_perftest() {
 		if [ $with_mpi -eq 1 ]
 		then
 			# Run UCP performance test
+			which mpirun
```

Comment:
`command -v <cmd>` is the preferred way according to shellcheck (`which` is not a standard utility).

Reply:
Checked before and 'which' is present in all images that we use.

Reply:
Double checked - looks like shellcheck now doesn't raise this warning.